bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -2343,18 +2345,22 @@ void ReplicaImp::onMessage<CheckpointMsg>(CheckpointMsg *msg) {
       // <= to allow repeating checkpoint message since state transfer may not kick in when we are inside active
       // window
       if (pos != tableOfStableCheckpoints.end()) delete pos->second;
-      CheckpointMsg *x = new CheckpointMsg(msgGenReplicaId, msgSeqNum, msgState, stateDigest, otherDigest, msgIsStable);
+      CheckpointMsg *x = new CheckpointMsg(
```

Comment:
should we rename x to stableChkPtMsg. 'x' is not giving any information about the variable.

Reply:
I did, thanks